### DISCUSSION TITLE
How to run the triton server cpu only on kubernetes?

### DISCUSSION THREAD

**Discussion by AlexXi19** [triton-inference-server/server#5492] [Q&A] [answered]
I'm trying to run a triton server on a local k8s cluster (m1) and am running into

```
E0313 05:42:56.104148 1 model_repository_manager.cc:1927] Poll failed for model directory 'add_sub': unable to get number of CUDA devices: unknown error
I0313 05:42:56.104177 1 model_repository_manager.cc:549] VersionStates() 'add_sub'
```

There are docs for running triton on cpu for docker but not for kubernetes. 

I just need a test model for sanity check purposes, is there a quick way to get this working?

**Comment by the-david-oy** [comment, marked as answer]
This issue has been seen before for CPU-only builds on M1 (M1 is not officially supported), with some potential answers provided [here](https://github.com/triton-inference-server/server/issues/3980).

**Reply by AlexXi19** [reply]
I will take a look, thanks for the help!
